client/src/styles/_bootstrap-variables.scss

```diff
@@ -79,12 +77,12 @@ $blue: #005a93; // custom
 $indigo:  #6610f2 !default; // bootstrap default
 $purple:  #6f42c1 !default; // bootstrap default
 $pink:    #e83e8c !default; // bootstrap default
-$red: #d40404; // custom
+$red:     #DA273B !default; // custom
```

Comment:
mix of upper case colours and lower case colours, could we stick to one for consistency? ;)

Reply:
Yup easy fix, typically all uppercase is cleaner.

Reply:
Also the spacing between variables and their values.

Reply:
Seems Bootstrap use all lower so best to stick to that actually :)